### DISCUSSION TITLE
Github Add Collaborator Bug

### DISCUSSION THREAD

**Discussion by SoberishAU** [community/community#63341] [Repositories]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Hi ive run into a bit of a strange bug regarding adding Github collaborators.
Ive been sent an invitation less then 12 hrs ago and its already unavailable/expired and presents a 404 page. 

Ive tried diagnosing the issue and i can tell that the problem is not relevant to my browser,
to my ISP server or authentication with me being logged into my account 

Ive attatched a gif to present the issue but i have no idea what is going on.
As the request says invitations should expire in 7 days where this has ended prematurely...

![GitHubBUg](https://github.com/community/community/assets/10396673/0d712dd9-fb40-4ad5-85e1-d549e02d7752)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @monarchshield ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
